docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,138 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+## Diagnostic and Recovery Tools
+
+### **WARNING : Please be careful and use this tool under supervision of DAOS export.**
+
+In case of PMEM device restored to healthy state, the ext4 filesystem
+created on each PMEM devices may need to verified and repaired if needed.
+
+### **Make sure that PMEM is not in use and not mounted while doing check or repair.**
+
+#### Use dmg command to stop the daos engines.
+```
+# dmg -o test_daos_dmg.yaml system stop
+Rank  Operation Result
+----  --------- ------
+[0-1] stop      OK
+```
+#### Stop the daos_server on the node where PMem checkking is required.
```

Comment:
I don't think that we should be using sub-section like this to list instruction. sub-sections with long title do not work well in our materials template.

Reply:
Reduce the title